Lib/tkinter/ttk.py

```diff
@@ -570,6 +571,15 @@ def state(self, statespec=None):
         return self.tk.splitlist(str(self.tk.call(self._w, "state", statespec)))
 
 
+    def cget(self, key):
+        """Return the resource value for a KEY given as string."""
+        if isinstance(return_value := super().cget(key), _tkinter.Tcl_Obj):
+            return str(return_value)
+        return return_value
```

Comment:
Instead of a complex test like that, I think it's just easier to do `return str(super().cget(key))`.

Reply:
Oh ok, it's the key that is a string, not the value. NVM.